scripts/verify-lint

```diff
@@ -55,4 +55,15 @@ echo "+++ Running spell check"
   "!content/en/v1.1-docs/reference/api-docs.md" \
   "!content/en/v1.2-docs/reference/api-docs.md" \
   "!content/en/v1.3-docs/reference/api-docs.md" \
-  "!content/en/next-docs/reference/api-docs.md"
+  "!content/en/next-docs/reference/api-docs.md" \
+  "!content/en/next-docs/**/*.md"
```

Comment:
Nit: If we have line 59, can we then remove line 58 since `content/en/next-docs/**/*.md` would already include that?

Reply:
Good point 😅 Fixed in 09f3762